packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -313,6 +313,28 @@ export function findSelectTokenButton(
     .should('have.text', text)
 }
 
+export function findTransactionInTransactionHistory({
+  text,
+  symbol,
+  amount
+}: {
+  text?: string
+  symbol: string
+  amount: number
+}) {
+  if (text) {
+    cy.findByText(text).should('be.visible')
+  }
+
+  return cy
```

Comment:
we should assert that "see details" button is present

Reply:
Added